.php_cs

```diff
@@ -1,32 +1,41 @@
 <?php
 
-if (class_exists('PhpCsFixer\Finder')) {    // PHP-CS-Fixer 2.x
+if (class_exists('PhpCsFixer\Finder')) {
+    echo "PHP-CS-Fixer 2.x rules\n";
     $finder = PhpCsFixer\Finder::create()
-        ->exclude('szef/')
-        ->exclude('templates/colorbox/')
-        ->exclude('templates/compile/')
-        ->exclude('templates/rating/')
-        ->exclude('templates/jpgraph/')
-        ->exclude('templates/libraries/')
-        ->exclude('templates/htmlpurifier/')
-        ->exclude('templates/sentry-php-master/')
-        ->exclude('templates/piwik-php-tracker/')
-        ->notPath('templates/GoogleMap.php')
-        ->notPath('templates/PasswordHash.php')
-        ->notPath('templates/JSMin.php')
-        ->notPath('szef/smarty.php')
-        ->in('website/')
+        ->exclude('website/szef/')
+        ->exclude('website/templates/colorbox/')
+        ->exclude('website/templates/compile/')
```

Comment:
The phpcs rules fix should be along with the fixes in commit f0356ce. So we have each commit "working". It could be useful one day if we want to `git bisect`…

I've this article one hour ago ;) https://disic.github.io/politique-de-contribution-open-source/pratique/#bonnes-pratiques-de-d%C3%A9veloppement

> Ne pousser que des commits de code qui compilent, testés et fonctionnels, accompagnés des tests unitaires correspondants ; certaines plateformes offrent la possibilité de rejouer automatiquement les tests unitaires d’un projet afin d’assurer la non-régression (e.g Travis, Homu)